### PR TITLE
Security Policy | Policy Objects | Zones and FQDN list

### PR DESCRIPTION
**Pull request by narasing01**
draft pull request

### CODE REVIEW

File: sdwan_policy_objects.tf [hunk 1]
```diff
@@ -175,3 +175,25 @@ resource "sdwan_region_list_policy_object" "region_list_policy_object" {
...
+  )
+}
+
+resource "sdwan_data_fqdn_prefix_list_policy_object" "fqdn_prefix_list_policy_object" {
+  for_each = { for p in try(local.policy_objects.fqdn_data_prefix_lists, {}) : p.name => p }
+  name     = each.value.name
+  entries = [for e in try(each.value.fqdns, []) : {
+    fqdn = e
+  }]
+
```
Comment (rrahimm) [inline sdwan_policy_objects.tf:198]: please remove additional spaces

**Review by rrahimm — CHANGES_REQUESTED** [review]
please remove additional spaces

**Review by rrahimm — APPROVED** [review]
tested good

**Review by rrahimm — APPROVED** [review]

**Comment by rrahimm** [comment]
Hello @danischm 

Kindly review and merge.

**Comment by narasing01** [comment]
duplicate of https://github.com/netascode/terraform-sdwan-nac-sdwan/pull/41

**Pull request closed without merging**